app/views/content/teacher-training-advisers.md

```diff
@@ -58,7 +58,7 @@ They're all former teachers with lots of experience, so they know the process in
 
 $q-tta-one$
 
-Wherever you are in your journey into teaching, an adviser can give you as much or as little help as you need. 
+An adviser can give you as much or as little help as you need. 
```

Comment:
I think this page is fine re these changes - but we do need to revisit it re the qual research suggestions!